### ISSUE TITLE
No flask-tracking.db file in part-0

### ISSUE THREAD

**Issue by pwolter** [mjhea0/flask-tracking#8]
Hi,

Looks like the flask-tracking.db file is missing from the part-0 repo?

I am trying to follow: https://realpython.com/python-web-applications-with-flask-part-i/

The directory tree shows it:

├── flask-tracking.db
├── requirements.txt
├── templates
│   ├── data_list.html
│   ├── helpers
│   │   ├── forms.html
│   │   └── tables.html
│   ├── index.html
│   ├── layout.html
│   └── validation_error.html
└── tracking.py

**Comment by svieira** [comment]
Yes, it should get created when you run the application for the first time - e. g. `your-virtual-env$ python3 tracking.py`
